apps/trip_plan/lib/trip_plan/api/open_trip_planner.ex

```diff
@@ -86,55 +84,133 @@ defmodule TripPlan.Api.OpenTripPlanner do
     Util.config(:trip_plan, OpenTripPlanner, key)
   end
 
-  defp send_request(url, params, parser) do
-    with {:ok, response} <- log_response(url, params),
-         %{status_code: 200, body: body} <- response do
+  defp send_request(url, query, parser) do
+    with {:ok, response} <- log_response(url, query),
+         %{status: 200, body: body} <- response do
       parser.(body)
     else
-      %{status_code: _} = response ->
+      %{status: _} = response ->
         {:error, response}
 
       error ->
         error
     end
   end
 
-  defp log_response(url, params) do
+  defp log_response(url, query) do
+    graphql_req = Req.new(base_url: url) |> AbsintheClient.attach()
+
     {duration, response} =
       :timer.tc(
-        HTTPoison,
-        :get,
-        [url, build_headers(config(:wiremock_proxy)), [params: params, recv_timeout: 10_000]]
+        Req,
+        :post,
+        [graphql_req, [graphql: query]]
       )
 
     _ =
       Logger.info(fn ->
-        "#{__MODULE__}.plan_response url=#{url} is_otp2=#{String.contains?(url, config(:otp2_url))} params=#{inspect(params)} #{status_text(response)} duration=#{duration / :timer.seconds(1)}"
+        "#{__MODULE__}.plan_response url=#{url} is_otp2=#{String.contains?(url, config(:otp2_url))} query=#{inspect(query)} #{status_text(response)} duration=#{duration / :timer.seconds(1)}"
       end)
 
     response
   end
 
-  defp build_headers("true") do
-    proxy_url = Application.get_env(:trip_plan, OpenTripPlanner)[:wiremock_proxy_url]
-    [{"X-WM-Proxy-Url", proxy_url}]
```

Comment:
@thecristen: Do we still need this header?

Reply:
Let's keep it; I'm not ready to give up on Wiremock (Java app that can record and save API responses for testing) yet despite the fact we don't rely on it at the moment...